apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -104,9 +104,9 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
         </div>
       </div>
       {!shouldLockDisableProps("apps").disabled && (
-        <div>
+        <div className="rounded-md bg-gray-50 p-8">
```

Comment:
we have our own color tokens, see bg-muted, bg-subtle that represent both light and dark mode. do not use bg-gray etc. unless in very rare edge cases and always have a dark color as well.